### PR TITLE
Node test

### PR DESCRIPTION
**Pull request by EdNoepel**
Takes ~~two~~ three arguments, node URI, account address, and key.  Used to submit small test transactions and test that a node supports the facilities our keeper framework requires.

### CODE REVIEW

**Comment by EdNoepel** [comment]
@brianmcmichael I'd like to include this test for others who may want to test node compatibility.  Your review would be appreciated.

File: tests/manual_test_node.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+from pymaker.numeric import Wad
+
+logging.basicConfig(format='%(asctime)-15s %(levelname)-8s %(message)s', level=logging.DEBUG)
+# reduce logspew
+logging.getLogger('urllib3').setLevel(logging.INFO)
+logging.getLogger("web3").setLevel(logging.INFO)
+logging.getLogger("asyncio").setLevel(logging.INFO)
+logging.getLogger("requests").setLevel(logging.INFO)
+
+endpoint_uri = f"https://localhost:8545"
```
[the remark this answers is not in the document]
Comment (brianmcmichael) [reply tests/manual_test_node.py:35]: This was a quick script Ed developed for me to test against other endpoints. I ended up manually testing against geth and infura by modifying this line. An arg would be nice (with default to localhost:8545), but not sure that this script will get much usage.

[the remark this answers is not in the document]
Comment (godsflaw) [reply tests/manual_test_node.py:35]: I'm fine either way.  I figured this was a one-off.

[the remark this answers is not in the document]
Comment (EdNoepel) [reply tests/manual_test_node.py]: I agree with @godsflaw ; the node URI should be an argument.

Also tacked on two other changes:
- unwrap the ETH wrapped for testing
- query past frobs to ensure retrieval of event logs works

**Pull request merged**